aeroflexSup/aeroflex_common.db

```diff
@@ -17,6 +17,27 @@ record(bo, "$(P)DISABLE")
     field(ONAM, "COMMS DISABLED")
 }
 
+record(bi, "$(P)CMODE_RBV") {
+    field(DTYP, "Soft Channel")
+    field(VAL, "$(DISABLE=0)")
+    field(SCAN, "Passive")
+    field(INP, "$(P)CMODE:SP CP")
+    field(ZNAM, "REMOTE")
+    field(ONAM, "LOCAL")
+}
+
+record(bo, "$(P)CMODE:SP") 
+{
+    field(DTYP, "Raw Soft Channel")
+    field(DESC, "Enable Local or Remote Mode")
+    field(ZRST, "REMOTE")
+    field(ONST, "LOCAL")
+	field(ZRVL, "0")
+	field(ONVL, "1")  
```

Comment:
Does this load OK? I think bo used ZNAM/ONAM instead

Reply:
Changed to ZNAM and ONAM.